src/Sql/Platform/Sqlite/SelectDecorator.php

```diff
@@ -55,6 +55,43 @@ protected function processStatementStart(
         return '';
     }
 
+    protected function processLimit(
+        PlatformInterface $platform,
+        DriverInterface $driver = null,
+        ParameterContainer $parameterContainer = null
+    ) {
+        if ($this->limit === null && $this->offset !== null) {
+            return [''];
+        }
+        if ($this->limit === null) {
+            return;
+        }
+        if ($parameterContainer) {
+            $paramPrefix = $this->processInfo['paramPrefix'];
+            $parameterContainer->offsetSet($paramPrefix . 'limit', $this->limit, ParameterContainer::TYPE_INTEGER);
+            return [$driver->formatParameterName('limit')];
+        }
+
+        return [$this->limit];
+    }
+
+    protected function processOffset(
+        PlatformInterface $platform,
+        DriverInterface $driver = null,
+        ParameterContainer $parameterContainer = null
+    ) {
+        if ($this->offset === null) {
+            return;
+        }
+        if ($parameterContainer) {
+            $paramPrefix = $this->processInfo['paramPrefix'];
+            $parameterContainer->offsetSet($paramPrefix . 'offset', $this->offset, ParameterContainer::TYPE_INTEGER);
+            return [$driver->formatParameterName('offset')];
+        }
+
+        return [$this->offset];
+    }
```

Comment:
I'd just to note that these methods are exactly the same as in:
https://github.com/zendframework/zend-db/blob/4675b97a5991279313818145538b70558419e410/src/Sql/Platform/Mysql/SelectDecorator.php#L41-L76

and the difference with parent class `Select` is to not quote offset and limit values.
I think it's fine, for MySQL it was added in that commit: https://github.com/zendframework/zend-db/commit/91bfba7944e36702c79e9acba7ebd7022b9a4611